src/Common/ExtensionMethods.cs

```diff
@@ -23,7 +23,7 @@ namespace Nvidia.Clara.DicomAdapter.Common
 {
     public static class ExtensionMethods
     {
-        public const int CLARA_JOB_NAME_MAX_LENGTH = 25;
+        public const int CLARA_JOB_NAME_MAX_LENGTH = 253;
         private static readonly Regex ValidJobNameRegex = new Regex("[^a-zA-Z0-9-]");
```

Comment:
this regex confuses me  the "Valid Job Name" is any single character which is NOT alpha-numeric and not a hyphen?

I'm thinking that you want something more like `"^[A-Z0-9a-z\\-]{1," + CLARA_JOB_NAME_MAX_LENGTH +"}$"`

Reply:
The regex contains valid characters for a job name.  Renamed to `ValidJobNameCharactersRegex `